src/analytics/private/metrics-common-workflow.ts

```diff
@@ -80,6 +85,16 @@ export function buildMetricsWidgetForWorkflows(scope: Construct, id: string, pro
   });
   (upsertUsersWorkflowAlarm.node.defaultChild as CfnResource).addPropertyOverride('Period', upsertUsersInterval.getIntervalSeconds());
 
+  const scanMetadataWorkflowAlarm = new Alarm(scope, id + 'ScanMetadataWorkflowAlarm', {
+    comparisonOperator: ComparisonOperator.GREATER_THAN_OR_EQUAL_TO_THRESHOLD,
+    threshold: 1,
+    evaluationPeriods: 1,
+    treatMissingData: TreatMissingData.NOT_BREACHING,
+    metric: props.scanMetadataWorkflow.metricFailed({ period: Duration.hours(24) }),
+    alarmDescription: `Scan metadata workflow failed, projectId: ${props.projectId}`,
+    alarmName: getAlarmName(scope, props.projectId, 'Scan Metadata Workflow'),
+  });
+  (scanMetadataWorkflowAlarm.node.defaultChild as CfnResource).addPropertyOverride('Period', processingJobInterval.getIntervalSeconds());
```

Comment:
Will it work if the processing interval is 2 days? @amliuyong

Reply:
it is handled correctly by custom resource `getIntervalSeconds()`

```javascript
  // Metrics cannot be checked across more than a day (EvaluationPeriods * Period must be <= 86400)
  if (intervalSeconds * evaluationPeriods > 86400) {
    intervalSeconds = 86400 / evaluationPeriods;
  }

```